diff --git a/node_modules/@cucumber/cucumber/lib/cli/argv_parser.js b/node_modules/@cucumber/cucumber/lib/cli/argv_parser.js
index f744a7b..e871248 100644
--- a/node_modules/@cucumber/cucumber/lib/cli/argv_parser.js
+++ b/node_modules/@cucumber/cucumber/lib/cli/argv_parser.js
@@ -57,6 +57,7 @@ const ArgvParser = {
             .usage('[options] [<GLOB|DIR|FILE[:LINE]>...]')
             .version(version, '-v, --version')
             .option('-b, --backtrace', 'show full backtrace for errors')
+            .option('-c, --configuration', 'for detox', '')
+            .option('--headless', 'for detox', '')
             .option('-d, --dry-run', 'invoke formatters without executing steps', false)
             .option('--exit', 'force shutdown of the event loop when the test run has finished: cucumber will call process.exit', false)
             .option('--fail-fast', 'abort the run on first failure', false)
